.github/workflows/integration.yml

```diff
@@ -59,3 +63,7 @@ jobs:
         # In future we'll be able to use the `--all` flag here to remove envs for
         # all Python versions (https://github.com/python-poetry/poetry/issues/3208).
         run: poetry env remove python3.8
+
+      - name: Remove Docker containers
```

Comment:
@zakaprov as discussed, just one cleanup - before the tests - should be enough, right?